libcst/codemod/_cli.py

```diff
@@ -616,171 +622,68 @@ def parallel_exec_transform_with_prettyprint(  # noqa: C901
+                if python_version is not None
+                else PartialParserConfig()
+            ),
         )
-        if isinstance(result.transform_result, TransformFailure):
-            return ParallelTransformResult(
-                successes=0,
-                failures=1,
-                skips=0,
-                warnings=len(result.transform_result.warning_messages),
-            )
-        elif isinstance(result.transform_result, (TransformSkip, TransformExit)):
-            return ParallelTransformResult(
-                successes=0,
-                failures=0,
-                skips=1,
-                warnings=len(result.transform_result.warning_messages),
-            )
-        elif isinstance(result.transform_result, TransformSuccess):
-            return ParallelTransformResult(
-                successes=1,
-                failures=0,
-                skips=0,
-                warnings=len(result.transform_result.warning_messages),
-            )
-        else:
-            raise Exception("Logic error, unaccounted for result!")
-
-    # Warm the parser, pre-fork.
-    parse_module(
-        "",
-        config=(
-            PartialParserConfig(python_version=python_version)
-            if python_version is not None
-            else PartialParserConfig()
-        ),
-    )
 
-    # Complex case, more than one file
     successes: int = 0
     failures: int = 0
     warnings: int = 0
     skips: int = 0
-    pending_processes: List[Process] = []
-
-    # Start processes
-    filename_to_process: Dict[str, Process] = {}
-    for f in files:
-        process = Process(
-            target=_parallel_exec_process_stub,
-            args=(
-                queue,
-                transform,
-                f,
-                repo_root,
-                unified_diff,
-                include_generated,
-                generated_code_marker,
-                format_code,
-                formatter_args,
-                blacklist_patterns,
-                python_version,
-            ),
-        )
-        pending_processes.append(process)
-        filename_to_process[f] = process
-
-    # Start the processes, allowing no more than num_processes to be running
-    # at once.
-    results_left = len(pending_processes)
-    joinable_processes: Set[Process] = set()
-    processes_started = 0
-
-    interrupted = False
-    while results_left > 0 and not interrupted:
-        while processes_started < jobs and pending_processes:
-            try:
-                # Move this process to the joinables
-                process = pending_processes.pop(0)
-                joinable_processes.add(process)
-
-                # Start it, bookkeep that we did
-                process.start()
-                processes_started += 1
-            except KeyboardInterrupt:
-                interrupted = True
-                continue
 
+    with pool_impl(processes=jobs) as p:  # type: ignore
```

Comment:
Hmm what is this type ignore suppressing?

Reply:
```
`Union[typing.Callable(Pool)[[Named(processes, Optional[int], default), Named(initializer, Optional[typing.Callable[..., typing.Any]], default), Named(initargs, typing.Iterable[typing.Any], default), Named(maxtasksperchild, Optional[int], default)], multiprocessing.pool.Pool], typing.Callable(type.__call__)[[Variable(typing.Any), Keywords(typing.Any)], typing.Any]]` is not a function.
```